app/src/lib/editors/darwin.ts

```diff
@@ -144,6 +144,10 @@ const editors: IDarwinExternalEditor[] = [
     name: 'Lite XL',
     bundleIdentifiers: ['com.lite-xl'],
   },
+  {
+    name: 'Spyder',
```

Comment:
happy to remove this if you wish to use [PR 15505](https://github.com/desktop/desktop/pull/15505)